packages/@lwc/engine/src/framework/invoker.ts

```diff
@@ -124,6 +124,7 @@ export function invokeComponentRenderMethod(vm: VM): VNodes {
         () => {
             // job
             const html = callHook(component, render);
+            isRendering = false;
```

Comment:
yeah, feel free to rename it.

Reply:
this line can't be done here... because if the render() method throws, the state is never restored! also inception is not preserved when you assign the false...